tests/run-tests.py

```diff
@@ -232,6 +233,11 @@ def send_get(what):
                     with open(test_file, "rb") as f:
                         # instead of: output_mupy = subprocess.check_output(args, stdin=f)
                         master, slave = pty.openpty()
+                        # disable auto newline conversion on the pty terminal.
+                        attrs = termios.tcgetattr(slave)
```

Comment:
You're probably aware but this doesn't exist on windows. Isn't there a way to just do a `replace("/r/n", "/n")` before comparing results, or something like that?

Reply:
Oh, I thought `pty` itself wasn't avaialble either so it would be a moot point? 
I initially thought of just adding _another_ newline replace, but thought fixing the problem at the source was cleaner than further fiddling with test results.

Reply:
Yeah it looks like these repl_* tests are already skipped on windows master builds https://ci.appveyor.com/project/dpgeorge/micropython/builds/48536058/job/4o16bx3lnvsu7nbn
![image](https://github.com/micropython/micropython/assets/3318786/4de4f577-4ffb-43f1-b309-6e5cfd4b02cc)